jcloud/normalize.py

```diff
@@ -39,6 +39,36 @@ class FlowYamlNotFound(FileNotFoundError):
     pass
 
 
+def stringify(v: Any) -> str:
+    if isinstance(v, str):
+        return v
+    elif isinstance(v, int) or isinstance(v, float):
+        return str(v)
```

Comment:
The value type is Any, but your coditions only have int/float and str. maybe you can print the error for other types?